# HG changeset patch
# User Thorsten Meyer <thorsten.meyier@gmx.de>
# Date 1233517210 -3600
#      Sun Feb 01 20:40:10 2009 +0100
# Node ID e50b44b7589155edd047c49f73f918d0c9214389
# Parent  ff61b53eb294138add7d31365ac1444324e4dc04
Remove printing of cvs motd in autogen.sh

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,12 @@
+2009-02-01  Thorsten Meyer  <thorsten.meyier@gmx.de>
+
+	* autogen.sh: Remove printing of cvs motd.
+
 2009-01-30  Marco Atzeri  <marco_atzeri@yahoo.it>
 
 	* configure.in (SHLLIBPRE, SHLBINPRE): Fix definitions for Cygwin.
 	(SHLEXT, SHLLIB, SHLBIN, SONAME_FLAGS): Fix definitions for Cygwin
 	and MinGW.
 
 2009-01-30  Benjamin Lindner  <lindnerb@users.sourceforge.net>
 
diff --git a/autogen.sh b/autogen.sh
--- a/autogen.sh
+++ b/autogen.sh
@@ -34,15 +34,8 @@ for i in `find . -name configure.in -pri
     if [ -f skip-autoheader ]; then
       echo "skipping autoheader in $dir"
     else
       autoheader --force
     fi
 ); done
 
 echo done
-
-if [ -f cvs.motd ]; then
-  echo "ATTENTION CVS Users!"
-  echo ""
-  cat cvs.motd
-  echo ""
-fi
